feature-libs/product/configurators/rulebased/cpq/cpq-configurator-serializer.spec.ts

```diff
@@ -7,13 +7,15 @@ import { Configurator } from './../core/model/configurator.model';
 const configId = '1';
 const attrCodeFirst = 111;
 const attrCode = 222;
-const attributeName = 'AttrName';
+const attributeName = '9999';
 const selectedSingleValue = 'SomeValue';
 const value1: Configurator.Value = { valueCode: 'ValueCode1', selected: true };
 const value2: Configurator.Value = { valueCode: 'ValueCode2', selected: false };
 const value3: Configurator.Value = { valueCode: 'ValueCode3', selected: true };
 const userInput = '123';
 const expectedValueIdsMulti = 'ValueCode1,ValueCode3,';
+const expectedValueIdsMultiNoSelection = ',';
+const expectedProcessedSingleValueNoValue = ',';
```

Comment:
expecting just a comma, when no value is selected seems odd to me?

Reply:
This is exactly the value which is expected from the CPQ update API to remove all / last values. Other values (e.g. empty string) leads to the "bad request".